spec/worker_spec.js

```diff
@@ -0,0 +1,30 @@
+var noop = require("lodash.noop");
+
+describe("worker", function() {
+  it("should handle message correctly", function() {
+    var fakeJasmine = jasmine.createSpyObj("fakeJasmine", ["onComplete"]),
+        fakeNewJasmine = jasmine.createSpy("fakeNewJasmine").and.returnValue(fakeJasmine),
+        fakeRunJasmine = jasmine.createSpy("fakeRunJasmine"),
+        fakeRequire = {
+          "./new": fakeNewJasmine,
+          "./run": fakeRunJasmine
+        },
+        env = {};
+    
+    spyOn(require("module"), "_load").and.callFake(function() {
+      if (fakeRequire[arguments[0]]) {
+        return fakeRequire[arguments[0]];
+      } else {
+        return this._load.and.originalFn.apply(this, arguments);
+      }
+    });
+
+    process.removeAllListeners("message");
+    require("../lib/worker.js")();
```

Comment:
Even with a single spec, we should still pull the `require` to the top of the file.

Reply:
I cannot. Same reason for the require spy. If I do not do it after the spy, I am unable to test the call.